README.md

```diff
@@ -18,7 +18,65 @@ with Bazel.
 
 ## Setup
 
-To be documented.
+### Using npm and Node.js rules
+
+We recommend you use npm to install the PostCSS build rules if you use the
+[Node.js build rules][rules_nodejs], and **strongly recommend** using npm
+if you also depend or plan to depend on PostCSS in your local `package.json`.
+(For example, if PostCSS plugin(s) will form part of your repository's build
+pipeline.)
+
+Installing the PostCSS build rules via npm allows you to rely on its native
+package management functionality. If you depend on PostCSS in your local
+`package.json`, using npm also helps avoid version skew between PostCSS in
+your repository and these build rules.
+
+[rules_nodejs]: https://bazelbuild.github.io/rules_nodejs/
+
+Add the `@bazel/postcss` npm package to your `devDependencies` in
+`package.json`.
+
+Your `WORKSPACE` should declare a `yarn_install` or `npm_install` rule named
+`npm`. It should then install the rules found in the npm packages using the
+`install_bazel_dependencies' function.
+See https://github.com/bazelbuild/rules_nodejs/#quickstart
+
+This causes the `@bazel/postcss` package to be installed as a Bazel workspace
+named `build_bazel_rules_postcss`.
```

Comment:
in 2.0 we now load just from `@npm//@bazel/postcss:index.bzl` there is no additional workspace

Reply:
2.0-rc.0 was shipped yesterday FYI

Reply:
Thanks!

Reply:
@alexeagle On upgrading to 2.0.0 my test repo doesn't seem to work anymore:

    load("@npm//@bazel/postcss:defs.bzl", "postcss_binary", "postcss_plugin")

Gets me:

    ERROR: error loading package '': cannot load '@npm//@bazel/postcss:defs.bzl': no such file

See https://gist.github.com/rzhw/d8217d54d74d23bc354789d72caf68a7

Reply:
Resolved.